server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/server.js

```diff
@@ -61,67 +61,60 @@ function refreshDetailTable() {
  * Generates the server history table
  */
 function refreshHistoryTable() {
-
-  clearTableBody('opHistoryDetails');
-
   var data = sessionStorage.server === undefined ?
       [] : JSON.parse(sessionStorage.server);
 
   if (data.length === 0 || data.allTimeTabletResults === undefined) {
-    var row = [];
-
-    row.push(createEmptyRow(8, 'Empty'));
-
-    $('<tr/>', {
-      html: row.join('')
-    }).appendTo('#opHistoryDetails tbody');
+    $("#opHistoryDetails > tbody > tr > td").each(function () {
+        $(this).text(0);
+    });
```

Comment:
This puts 0 in every cell. I think its OK since this is only if no data comes back from the REST call. But I am not sure of when this actually happens.

Reply:
Should you just put `""` then?